lib/Fhp/Action/GetSepaDirectDebitLeadTime.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Fhp\Action;
+
+use Fhp\BaseAction;
+use Fhp\Segment\DME\HIDMESv1;
+use Fhp\Segment\DME\HIDMESv2;
+use Fhp\Segment\DME\MinimaleVorlaufzeitSEPALastschrift;
+use Fhp\Segment\DSE\HIDSESv1;
+
+/**
+ * Retrieves information about lead times for SEPA Direct Debit Requests
+ */
+class GetSepaDirectDebitLeadTime extends BaseAction
+{
+    const SEQUENCE_TYPES = ['FRST', 'OOFF', 'FNAL', 'RCUR'];
+    const CORE_TYPES = ['CORE', 'COR1'];
+
+    /** @var MinimaleVorlaufzeitSEPALastschrift|null */
+    private $leadTime;
+
+    /** @var string */
+    private $coreType;
+
+    /** @var string */
+    private $seqType;
+
+    /** @var bool */
+    private $singleDirectDebit;
+
+    public static function create(string $seqType, bool $singleDirectDebit, string $coreType = 'CORE')
+    {
+        if (!in_array($coreType, self::CORE_TYPES)) {
+            throw new \InvalidArgumentException('Unknown CORE Type');
```

Comment:
Macht vermutlich Sinn, hier und unten den konkreten $coreType/$seqType in die Fehlermeldung einzubauen. Je nach dem, wie komplex die aufrufende Anwendung ist, weiß ihr Entwickler nicht so genau, was er hier reingibt.